### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by SnigdhaP2022**
## Created folders for B.com and B.Sc courses and added materials related to it 
This closes ISSUE #478 
Please provide a brief description of the changes introduced in this pull request. Explain the problem you are solving or the feature you are adding.
I have created folders for other courses and styled them according to the website's needs and specifications alignment. I have also related materials to the branches I have mentioned according to the course.
## Pull Request Checklist

Please make sure that your PR meets the following requirements:

- [✅ ] The app is working correctly after applying the changes.
- [ ✅] The app is responsive and displays properly on different screen sizes and devices.
- [✅ ] The changes are thoroughly documented in the appropriate files or documentation sections.
- [ ✅] The coding conventions and style guidelines are followed consistently.

## Checklist

Please check all applicable boxes and provide any additional information if needed.

- [✅ ] I have tested the changes locally and ensured that the app is functioning as expected.
- [ ✅] I have verified that the app is responsive and looks good on various screen sizes and devices.
- [ ✅] I have reviewed the documentation and ensured that it accurately reflects the changes made.
- [ ✅] I have followed the coding conventions and style guidelines defined in the project.
- [✅ ] I have run the automated tests and all of them pass.
- [ ✅] I have considered the performance implications of the changes made.
- [✅ ] I have considered the security implications of the changes made.
- [✅ ] I have checked for any potential regressions or side effects caused by the changes.
- [✅ ] I have performed a self-review of my code.
- [✅ ] Starred the repository.
- [ ] Added My Name to the Contributor's List.
- [ ✅] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [✅ ] I ran the app and tested it locally to verify that it works as expected.

# Screenshots - If Any (Optional)
Screenshot 2023-07-17 at 4 43 36 PM

Screenshot 2023-07-17 at 4 44 02 PM

## Additional Notes(Optional)
I have contributed under GSSOC'23. I am happy to be a part of this project. Thank you for the opportunity. Excited to see myself in the contribution list after merging.
Please provide any additional information or context that may be helpful for reviewing the pull request.

## Related Issues

Please list any related issues or pull requests that are associated with this pull request.

## Reviewers
@Harshil-Jani  Kindly, review this PR

Please tag the relevant team members or reviewers who should review this pull request.

### CODE REVIEW

**Comment by SnigdhaP2022** [comment]
@Harshil-Jani  Kindly, review this PR. Thank you for making me part of this project.

**Review by anmode — CHANGES_REQUESTED** [review]
can these jpg files convert into webp?? actually jpg take lot of size

**Comment by SnigdhaP2022** [comment]
Yeah, I'll try to make the necessary changes and commit the files.

**Comment by SnigdhaP2022** [comment]
@anmode  I have made requested changes by changing images jpg to webp. Kindly, review this and merge the files

**Comment by Harshil-Jani** [comment]
@SnigdhaP2022 Thanks for contributing to other courses. This helps us a lot. I would like to see a small enhancement into this and that is when from the Navbar you select on `B.Comm` or `BSc` then it should open up page just like it opens up in case of `B.Tech`. Can you please make this possible and we can merge this to help students from other streams ? 🎉

File: pages/courses/bcom/index.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import Header from "components/common/Header/Header";
+import styles from "@styles/Btech.module.css";
+import Footer from "components/common/Footer";
+
+export default function BTech() {
+  const image = "/images/btech/top_bg.jpg";
+  return (
+    <>
+      <Header image={image} text="B.Com" />
+      {/* <!-- content of b.tech section  --> */}
```
Comment (Harshil-Jani) [inline pages/courses/bcom/index.js]: update comment to change it for `b.com`

File: pages/courses/bsc/index.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import Header from "components/common/Header/Header";
+import styles from "@styles/Btech.module.css";
+import Footer from "components/common/Footer";
+
+export default function BTech() {
+  const image = "/images/btech/top_bg.jpg";
+  return (
+    <>
+      <Header image={image} text="B.Sc" />
+      {/* <!-- content of b.tech section  --> */}
```
Comment (Harshil-Jani) [inline pages/courses/bsc/index.js]: ditto here

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]

File: pages/courses/bcom/index.js (hunk 1 above)
Comment (SnigdhaP2022) [reply pages/courses/bcom/index.js]: Updated

File: pages/courses/bsc/index.js (hunk 2 above)
Comment (SnigdhaP2022) [reply pages/courses/bsc/index.js]: Updated

**Comment by SnigdhaP2022** [comment]
@Harshil-Jani  @anmode Requested changes have been updated. Kindly, review the file and merge the PR 🎉

**Comment by Harshil-Jani** [comment]
Thankyou for such valuable contribution @SnigdhaP2022 🎉

**Review by Harshil-Jani — APPROVED** [review]

**Comment by SnigdhaP2022** [comment]
@Harshil-Jani  requesting you to kindly add level label for the PR. Otherwise it won't consider for GSSOC'23 Contribution.

**Pull request merged**